docker/run_jormungandr.sh

```diff
@@ -1,14 +1,65 @@
 #!/usr/bin/env bash
 
+file=/usr/src/app/jormungandr.wsgi
+jormungandr_cache2="name=jormungandr,items=2048"
+monito_cache2="name=monitor,items=100"
+
+
+function show_help() {
+    cat << EOF
+Usage: ${0##*/} -j jormungandr-processes -m monitor-processes -r max-requests
+    -j      jormungandr-processes
```

Comment:
below, the naming is app_processes:
```suggestion
    -a      app-processes number
```